uberon_edit.obo

```diff
@@ -101992,22 +101992,46 @@ xref: UMLS:C0458765 {source="NIFSTD:birnlex_2555"}
 xref: UMLS:C0458764 {source="NIFSTD:birnlex_2557"}
 
 [Term]
+id: UBERON:0036250
+name: zone of basilar membrane of cochlea
+synonym: "zone of organ of corti" RELATED  []
+intersection_of: UBERON:0005764 ! acellular membrane
+intersection_of: part_of UBERON:0002226 ! basilar membrane of cochlea
+xref: SCTID:361065006 ! Zone of organ of Corti
+
+[Term]
 id: UBERON:0002820
 name: zona arcuata of basilar membrane of cochlea
-def: "The inner part of the basilar membrane of the cochlea that supports the spiral organ of Corti." [Wikipedia:Inner_tunnel]
+def: "The inner part of the basilar membrane of the cochlea that supports the spiral organ of Corti." [Wikipedia:Inner_tunnel, https://github.com/obophenotype/uberon/issues/1357]
 synonym: "zona arcuata" BROAD [Wikipedia:Inner_tunnel]
 synonym: "inner tunnel" BROAD [Wikipedia:Inner_tunnel]
 alt_id: UBERON:0002821
 alt_id: UBERON:0031464
 synonym: "basilar membrane of cochlea, zona arcuata" EXACT []
+synonym: "arcuate zone of organ of corti" EXACT []
+synonym: "arcuate zone of basilar membrane" EXACT []
 xref: FMA:75709 ! Zona arcuata of basilar membrane of cochlea
 xref: NIFSTD:birnlex_2528 ! Zona arcuata of basilar membrane of cochlea
-xref: NIFSTD:birnlex_2529 ! Zona arcuata of basilar membrane of cochlea
 is_a: UBERON:0005764 ! acellular membrane
 relationship: part_of UBERON:0002226 ! basilar membrane of cochlea
 xref: UMLS:C1184862 {source="NIFSTD:birnlex_2528"}
-xref: UMLS:C1184863
 xref: Wikipedia:Inner_tunnel ! inner tunnel
+xref: SCTID:368961005 ! Entire arcuate zone of organ of Corti
+
+[Term]
+id: UBERON:0036249
+name: zona pectinata of basilar membrane of cochlea
+def: "The acellular membrane that forms the outer two thirds of the basilar membrane of the conchlea." [http://medical-dictionary.thefreedictionary.com/zona+pectinata, https://github.com/obophenotype/uberon/issues/1357]
+synonym: "zona pectinata" BROAD []
+synonym: "pectinate zone" BROAD []
+synonym: "pectinate zone of organ of corti" EXACT []
+synonym: "pectinate zone of basilar membrane" EXACT []
+is_a: UBERON:0005764 ! acellular membrane
+relationship: part_of UBERON:0002226 ! basilar membrane of cochlea
+xref: NIFSTD:birnlex_2529 ! Zona arcuata of basilar membrane of cochlea
```

Comment:
The latest version fixes this so can use `NIFSTD:birnlex_2529 ! Zona pectinata of basilar membrane of cochlea`